quipucords/api/models.py

```diff
@@ -31,13 +31,24 @@ class HostCredential(Credential):
     sudo_password = models.CharField(max_length=1024, null=True)
     ssh_keyfile = models.CharField(max_length=1024, null=True)
 
-    # pylint: disable=arguments-differ
-    def save(self, *args, **kwargs):
+    def encrypt_fields(self):
+        """Encrypt the sensitive fields of the object"""
         if self.password:
             self.password = encrypt_data_as_unicode(self.password)
         if self.sudo_password:
             self.sudo_password = encrypt_data_as_unicode(self.sudo_password)
+
+    def save(self, *args, **kwargs):
+        """Save the model object
+        """
+        self.encrypt_fields()
         super(HostCredential, self).save(*args, **kwargs)
```

Comment:
Since we are Python 3 only, you can just use `super().save(...)`.

(https://docs.python.org/3/library/functions.html#super)